src/controllers/update/update.interrupt.card.controller.ts

```diff
@@ -0,0 +1,28 @@
+import { NextFunction, Request, Response } from "express";
+
+import { logger } from "../../utils/logger";
+import * as config from "../../config";
+
+export const get = (req: Request, res: Response, next: NextFunction) => {
+  try {
+    logger.debugRequest(req, `GET ${config.UPDATE_INTERRUPT_CARD_PAGE}`);
+
+    return res.render(config.UPDATE_INTERRUPT_CARD_PAGE, {
+      backLinkUrl: config.SECURE_UPDATE_FILTER_URL,
+      templateName: config.UPDATE_INTERRUPT_CARD_PAGE,
+    });
+  } catch (error) {
+    logger.errorRequest(req, error);
+    next(error);
+  }
+};
+
+export const post = (req: Request, res: Response, next: NextFunction) => {
+  try {
+    logger.debugRequest(req, `POST ${config.UPDATE_INTERRUPT_CARD_PAGE}`);
```

Comment:
consider changing this as we have done for our recent tickets (see check.your.answers.controller for an example):
logger.debugRequest(req, `${req.method} ${req.route.path}`);